File: Function_practise/01_random/random-3-1.js
// (3)	產生四個由數字和大寫英文字母組成

function random3() {
    let x = "";
    const characters = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ";

    for (let i = 0; i < 4; i++) {
        const index = Math.floor(Math.random() * characters.length);
        const n = characters[index];
        x += n;
    }

    return x;
}

console.log(random3());

File: Function_practise/01_random/random-3-1.py
# (3)	產生四個由數字和大寫英文字母組成

import random
def random_3():
    x = ""
    number = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ"
    for i in range(4):
        n = random.choice(number)
        x += str(n) 
    return x    
print(random_3())
